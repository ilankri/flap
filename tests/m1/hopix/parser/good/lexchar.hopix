val x = 'a'
val x = 'A'
val x = '*'
val x = '@'
val x = '\\'
val x = '\''
val x = '\n'
val x = '\t'
val x = '\b'
val x = '\r'
val x = '\000'
val x = '\001'
val x = '\255'
val x = '\0xFF'
val x = '\0XaB'
val x = '\0x12'
val x = '\0b11001100'
val x = '\0B11001100'
