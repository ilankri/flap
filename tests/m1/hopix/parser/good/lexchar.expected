
val x = 'a'
val x = 'A'
val x = '*'
val x = '@'
val x = '\\'
val x = '\''
val x = '\n'
val x = '\t'
val x = '\b'
val x = '\r'
val x = '\000'
val x = '\001'
val x = '\255'
val x = '\255'
val x = '\171'
val x = '\018'
val x = '\204'
val x = '\204'
